sdcm/cluster.py

```diff
@@ -2925,6 +2928,46 @@ def node_setup(_node):
                 cl_inst.log.error(msg)
                 raise NodeSetupTimeout(msg)
 
+        start_time = time.time()
```

Comment:
Since Python 3.3 it's better to use `time.perf_counter()`

Reply:
changed